dist/platforms/mac/steps/build.sh

```diff
@@ -19,6 +19,23 @@ echo "Using build name \"$BUILD_NAME\"."
 
 echo "Using build target \"$BUILD_TARGET\"."
 
+#
+# Display the build profile
+#
+
+if [ -z "$BUILD_PROFILE" ]; then
+  # User has not provided a build profile
+  #
+  echo "Doing a default \"$BUILD_TARGET\" platform build."
```

Comment:
This is already said on line 20.
Probably better to only add the echo from when `$BUILD_PROFILE` is set.